### PR TITLE
giftcard errors coverage

### PR DESCRIPTION
**Pull request by sathiyaaa**
## Purpose of this pull request

This pull request (PR) covers the error messages in the remove-giftcard mutation

## Affected DevDocs pages

-  src/guides/v2.3/graphql/mutations/remove-giftcard.html
- src/guides/v2.4/graphql/mutations/remove-giftcard.md

### CODE REVIEW

File: src/guides/v2.3/graphql/mutations/remove-giftcard.md [hunk 1]
```diff
@@ -77,3 +77,10 @@ Attribute |  Data Type | Description
  {% include graphql/cart-object.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find a cart with ID \"xxxxx\"` | The ID provided in the `cart_id` field is invalid or the cart does not exist for the customer.
+`The gift card couldn't be deleted from the quote.` | The value provided in the `gift_card_code` field is invalid or the giftCard with that specific ID does not exist in the cart.
```
Comment (BarnyShergold) [inline src/guides/v2.3/graphql/mutations/remove-giftcard.md:86]:
```suggestion
`The gift card couldn't be deleted from the quote.` | The value provided in the `gift_card_code` field is invalid or the gift card with that specific ID does not exist in the cart.
```

File: src/guides/v2.4/graphql/mutations/remove-giftcard.md [hunk 2]
```diff
@@ -69,3 +69,10 @@ Attribute |  Data Type | Description
  {% include graphql/cart-object-24.md %}
 
 [Cart query output]({{page.baseurl}}/graphql/queries/cart.html#cart-output) provides more information about the `Cart` object.
+
+## Errors
+
+Error | Description
+--- | ---
+`Could not find a cart with ID \"xxxxx\"` | The ID provided in the `cart_id` field is invalid or the cart does not exist for the customer.
+`The gift card couldn't be deleted from the quote.` | The value provided in the `gift_card_code` field is invalid or the giftCard with that specific ID does not exist in the cart.
```
Comment (BarnyShergold) [inline src/guides/v2.4/graphql/mutations/remove-giftcard.md:78]:
```suggestion
`The gift card couldn't be deleted from the quote.` | The value provided in the `gift_card_code` field is invalid or the gift card with that specific ID does not exist in the cart.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Just two small changes

**Review by BarnyShergold — APPROVED** [review]
Thanks!

**Review by keharper — APPROVED** [review]

**Pull request merged**